new/PRS14056DelormeE.xml

```diff
@@ -0,0 +1,58 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS14056DelormeE" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>M. Émile Delorme</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DR"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DR" when="2023-08-14">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:id="n1"><forename>M. Émile</forename> <surname>Delorme</surname></persName>
+                    <nationality type="France"/>
+                    <floruit when="1911">M. Émile Delorme, a French official, acquired manuscripts on a trip to Ethiopia in 1911. 
+                        His collection, partly described by <persName ref="PRS4805Grebaut"/>, later became dispersed, with several of his
```

Comment:
I do not know further details about him, but M. probably stands for "monsieur" only.

Reply:
I also found that a certain Émile Delorme was vice-president of the [Conseil d'administration de la Silocoop in Tunis in 1929](https://gallica.bnf.fr/ark:/12148/bpt6k51422954/f4.image.r=delorme) (could well be the same one, but of course not 100%), just a curiosity

Reply:
Great, thank you. 
Could you give me a reference for Walda Maryam? I do not find him in the sources mentioned here.

Reply:
I believe it was in a footnote in Grébaut

Reply:
![image](https://github.com/BetaMasaheft/Persons/assets/17987728/2d9b9ffc-6b19-490e-ac5d-e4804b7f88d6)